tabbycat/checkins/templates/CheckInStatusContainer.vue

```diff
@@ -233,6 +254,28 @@ export default {
       }
       return this.entitiesByTime
     },
+    entitiesByBreaking: function () {
+      const isBreaking = (entityByPresence) => {
+        if (entityByPresence.type === 'Adjudicator') {
+          return entityByPresence.breaking
+        } else if (this.breakingTeamShortNames.includes(entityByPresence.name)) {
+          return true
+        }
+        return false
+      }
+      if (this.filterByBreaking.All) {
+        return this.entitiesByPresence // No filter
+      } else if (this.filterByBreaking.Breaking) {
+        console.log('Breaking Teams and Adjudicators')
+        // print each object in the array
+        const temp = this.entitiesByPresence.filter(isBreaking)
+        for (let i = 0; i < temp.length; i++) {
+          console.log(temp[i])
+        }
```

Comment:
Avoid using `console`, as it's just for debugging.

Reply:
Yep, it's just temporary, will remove